src/playwright-api/makeTest.ts

```diff
@@ -86,7 +92,13 @@ export const makeTest = (
           viewports: [page.viewportSize().width],
         };
 
-        await writeTestResult(testInfo, snapshots, resourceArchive, chromaticStorybookParams);
+        await writeTestResult(
+          testInfo,
+          snapshots,
+          resourceArchive,
+          chromaticStorybookParams,
+          page.url()
```

Comment:
It looks like we're only passing this all the way through to be used in `ArchiveFile`. I think we could avoid needing this by just always saving files with the `/sanitized-domain/` preface. Would that work?

Reply:
Another option to avoid passing it through would be to include the source page in the [Archive  Response](https://github.com/chromaui/test-archiver/blob/dc822811d295bb3f62f262752c7e4eae99f4628a/src/resource-archive/index.ts#L9), which is provided to the `ArchiveFile`.

Reply:
@tevanoff you're right, we shouldn't need to pass this now! I'll update this afternoon.

Reply:
After looking this over, I don't think we want to always use the domain in the file path -- since the vast majority of users won't ever archive any cross-domain assets, we'd be bloating up the directory structure for a small number of users. 

We also wouldn't want to add the page URL to the `ArchiveResponse` since we'd store this for each archived response (we only need this value propagated at the test level).

We could make `resourceArchive` an object that has the page URL and the archive, but it would be odd to change its structure.

I'll keep it as-is for now, but happy to chat further if you see other ways of doing this.

Reply:
We don't store the full `ArchiveResponse`, we only write its `body` to disk, so it may be fine to duplicate it in there. 

Alternatively, we could make the `ResourceArchive` itself a full object with `sourcePage` and `archive` attributes.

Reply:
@tevanoff I ended up spreading it into `testInfo` which we pass into `writeTestArchive`. This worked nicely as it is indeed test info. I changed the type of `TestInfo` to be less Playwright-specific as well, which will be good for Cypress support too.

Reply:
Good call! 🙌